CleverPush/Source/CPChatView.m

```diff
@@ -270,6 +270,9 @@ - (void)userContentController:(WKUserContentController*)userContentController
         [CleverPush subscribe:^(NSString* subscriptionId) {
             // wait for ID
             [CleverPush getSubscriptionId:^(NSString* subscriptionId) {
+                if (subscriptionId == nil || [subscriptionId isKindOfClass:[NSNull class]] || [subscriptionId isEqualToString:@""]) {
```

Comment:
subscriptionId can not be NSNull or empty string. I think `nil` check should be enough in all spots. Do you agree @cpandya25 ? :)